src/components/common/SwapPrice.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta, Story } from '@storybook/react/types-6-0'
+import { SwapPrice, Props } from './SwapPrice'
+
+export default {
+  title: 'Common/SwapPrice',
```

Comment:
Book the saddest it's been

Reply:
This one was addressed in https://github.com/gnosis/dex-react/pull/1388